Week1/solution2.py

```diff
@@ -0,0 +1,6 @@
+#Write a python script to enter any number and print the sum of its digit.
+total=0
+n=int(input("Enter a number:"))
+for i in range(1,n+1):

```

Comment:
you understand wrongly this program definition.

You enter number eg. 123 then find sum of its digit 1+2+3=6.

please make a change.

Reply:
Sir, the program is working fine unless i have mistaken
Output example using IDLE- 
Enter a number:234
Total = 9